app/models/student.rb

```diff
@@ -22,6 +22,8 @@ class Student < User
 
   has_many :grace_period_deductions, through: :memberships
 
+  has_many :peer_reviews
```

Comment:
The association should be for groupings, not students.

Also you'll want two associations, one which is an inverse of `reviewer`, and another which is an inverse of `reviewee`